### PR TITLE
Update n3js and RDFLib and use RSS.

### PR DESCRIPTION
**Pull request by KonradHoeffner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,16 @@ type `make` in the root directory of the project
...
 
+### Further Requirements
+
+#### n3js
+
+    export NODE_OPTIONS=--max_old_space_size=16000
+
+#### librdf
+
+pacman -S redland
```
Comment (pchampin) [inline README.md:31]:
```suggestion
one of the following (depending on your distribution):

* pacman -S redland
* apt install librdf-dev
```

**Review by pchampin — APPROVED** [review]
looking good; I just suggested an addition to the README for installing redland/librdf

File: README.md (hunk 1 above)
Comment (KonradHoeffner) [reply README.md:31]: Oh yes, I forgot about Ubuntu, thanks for the addition.

**Comment by pchampin** [comment]
Sorry for the looong delay... I totally phased out this PR. Thanks for keeping this benchmark alive :sweat_smile: .

I'll probably get back to it when I release 0.8, which should be not too long, now...

**Pull request merged**